datadog/resource_datadog_dashboard.go

```diff
@@ -5157,9 +5176,9 @@ func buildDatadogApmStatsQuery(terraformQuery map[string]interface{}) *datadogV1
 	}
 
 	if terraformColumns, ok := terraformQuery["columns"].([]interface{}); ok && len(terraformColumns) > 0 {
-		datadogColumns := make([]string, len(terraformColumns))
+		datadogColumns := make([]map[string]interface{}, len(terraformColumns))
```

Comment:
I am new to Go. `columns` is an array of objects/dictionaries, i.e. `[{ "name": "foo" }, { "name": "bar" }]`. Is this the correct typing for that?

Reply:
Update: This is solved with a new type, `ApmStatsQueryColumnType`